assets/encode_url_base64.py

```diff
@@ -0,0 +1,9 @@
+import base64
+
+# String to encode
+text = "https://docs.google.com/forms/u/0/d/e/1FAIpQLScYvVDqRnRS3LfQMmz2yvt2sfGUbRhBevserv-0-67CnzKqMA/formResponse"
```

Comment:
Why providing the text to encrypt?

Reply:
True, should parse from the command line here. Was just doing this while setting up the script, will correct this!

Reply:
I know we're on the same track